### PR TITLE
Bug-fixes and minor improvements to end-of-match formspec

### PR DESCRIPTION
**Pull request by ClobberXD**
## Proposed changes
- Increase width of formspec, and space out labels to prevent clipping and overlapping.
- Improve match duration format. Add zero-padding and ':' separators
```
1h3m8s -> 01:03:08
44m1s  -> 00:44:01
```

### CODE REVIEW

File: mods/ctf_stats/gui.lua [hunk 1]
```diff
@@ -38,6 +38,10 @@ function ctf_stats.get_formspec_match_summary(stats, winner_team, winner_player,
 		blue.score = blue.score + pstat.score
 	end
 
+	local time_hours = string.format("%02d", math.floor(time/3600))
+	local time_minutes = string.format("%02d", math.floor((time % 3600) / 60))
+	local time_seconds = string.format("%02d", math.floor(time % 60))
```
Comment (rubenwardy) [inline mods/ctf_stats/gui.lua]:
```lua
local match_length = string.format("%02d:%02d:%02d",
		math.floor(time/3600), -- hours
		math.floor((time % 3600) / 60)), -- minutes
		math.floor(time % 60)) -- seconds
```

**Comment by ClobberXD** [comment]
Thanks for pointing that out. That's new to me :3

**Comment by rubenwardy** [comment]
Have you tested this? Not sure if `:` will need to be escaped

**Comment by ClobberXD** [comment]
I had tested this before the last commit, worked as expected. I'm sure it'll still work alright.

**Comment by ClobberXD** [comment]
@rubenwardy: Are there any more issues to fix? Please update your review. :)

**Pull request merged**